pyroomacoustics/transform/stft.py

```diff
@@ -44,7 +43,7 @@ class STFT(object):
         hop size
     analysis_window : numpy array
         window applied to block before analysis
-    synthesis : numpy array
+    synthesis_window : numpy array
```

Comment:
I agree with renaming the keyword for consistency, but did you check it doesn't break any other code/example ?

Reply:
Within the `__init__` constructor the keyword `synthesis_window` was consistently used already as follows
   ``` 
def __init__(self, N, hop=None, analysis_window=None,
                 synthesis_window=None, channels=1, transform='numpy',
                 streaming=True, precision='double', **kwargs):

        self.synthesis_window = synthesis_window
```

 so renaming only the docstring should not break other parts. Am I right?

Reply:
Oh, I missed this! Then no problem, nothing should be broken! 😄